student.rb

```diff
@@ -1,12 +1,19 @@
+require_relative 'classroom'
+
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+  def initialize(age, name = 'Unknown', parent_permission: true)
     super(name, age, parent_permission)
-    @classroom = classroom
+    @classroom = nil
   end
 
   def play_hooky
     '¯\\(ツ)/¯'
   end
+
+  def classrooms(classroom)
+    @classroom = classroom
+    classroom.add_student(self)
+  end
```

Comment:
- Please implement a custom setter that serves as a `belongs-to` relationship etween the `student` class and the `classroom` class. i.e
 ```ruby
    def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```